addons/point_of_sale/static/tests/tours/product_screen_tour.js

```diff
@@ -129,6 +129,13 @@ registry.category("web_tour.tours").add("ProductScreenTour", {
                 })
             ),
             ProductScreen.isShown(),
+
+            // Test Cancel Order from Actions
+            ProductScreen.clickReview(),
+            ProductScreen.clickControlButton("Cancel Order"),
+            Dialog.confirm(),
+            { ...ProductScreen.back(), isActive: ["mobile"] },
```

Comment:
```suggestion
```

remove this step.

Reply:
- In this tour we need this step as after this step we check if order is empty and that function triggeres review button if there is a mobile view
- so to have that button we have to be on productscreen instead of cart page otherwise this tour fails
![image](https://github.com/user-attachments/assets/cb578f4b-6d85-4d95-8f58-c3ec6e3a073e) 
![image](https://github.com/user-attachments/assets/aaa96612-f84c-483e-86cd-fcd58a2f7650)